### ISSUE TITLE
Ignore different non-custom types in ===

### ISSUE THREAD

**Issue by lakerka** [vitalets/babel-plugin-runtyper#9]
I want to add this library to existing project but when I run it I get tons of warnings related to `===` comparing non-custom types like string and int and so on. Is it possible to disable this type of comparison checking?

**Comment by vitalets** [comment]
hi @lakerka ! 
Do you mean you want to allow `1 === "1"`, `1 === true` etc?

**Comment by lakerka** [comment]
@vitalets yes. Currently there are so many warnings that it is impossible to use this tool. Filtering helps but it is still too much.

**Comment by vitalets** [comment]
But .. if you compare different types with strict equal, you will always get `false`:
```js
1 === "1" // => false
1 === true // => false
```

Could you show some example of code that shows warning?

**Comment by lakerka** [comment]
Unless you are using variables like: `a === b` during runtime they might get various values and some of them might have different types resulting in warning. In our codebase this happens very often therefore when we open console it explodes with warnings. So we want to start slow and first fix arithmetic and comparison errors and maybe later start fixing `===` issues.

**Comment by lakerka** [comment]
@vitalets I would like to open pull request how can I do that?

**Comment by vitalets** [comment]
It would be great! Just follow this [instruction](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).

**Comment by lakerka** [comment]
@vitalets I know there might be issues so please point out any required changes in pull request: https://github.com/vitalets/babel-plugin-runtyper/pull/10
